app/assets/stylesheets/cards.css

```diff
@@ -1,4 +1,50 @@
 /* Shared styles across pages for Bootstrap cards */
+
+/* Digital Content and Access Restriction cards share styles */
+#facet-digital_content,
+.online-contents,
+.digital-object-list {
+  --bs-card-border-width: 0;
+  --bs-card-border-radius: 0;
+  background-color: var(--stanford-fog-light);
+  filter: var(--drop-shadow-filter);
```

Comment:
If the intent is for the digital content card to have a drop shadow, I am not seeing that in the browser

<img width="1007" alt="Screenshot 2025-04-29 at 11 27 47 AM" src="https://github.com/user-attachments/assets/770af665-b2ab-44e1-b736-02371becb8e7" />

Reply:
Missing semi-colon: https://github.com/sul-dlss/stanford-arclight/compare/b0a8e2ca33405d2d40a21c415757d8b99a848682..b3f13454cb471c24eb60d66ec8d61925e99b6fdd

Thanks for catching that.